TextAnalysis/src/langanal/word/base/WordInfo.java

```diff
@@ -1,35 +1,51 @@
 package langanal.word.base;
 
 import javax.json.*;
+
+import langanal.word.base.Word;
+
 import java.util.LinkedList;
 import java.io.InputStream;
+import java.net.MalformedURLException;
 import java.net.URL;
 
 public class WordInfo {
 	private static final String thesaurusApiKey = "Jcglr2EapVZhu3ucPZsc"; //Api key used for online thesaurus resource
-	
-	
+
+	public static void main(String[] args){
+		for(Word w :getDictionaryWords("big")){
```

Comment:
What's the "big" here for? Seems like that should've been for testing or something but stuck around